pkg/util/ecs/metadata/v1/types.go

```diff
@@ -13,8 +13,9 @@ type Commands struct {
 // Instance represents the instance metadata exposed by the ECS introspection endpoint.
 // See http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-introspection.html
 type Instance struct {
-	Cluster string `json:"Cluster"`
-	Version string `json:"Version"`
+	Cluster              string `json:"Cluster"`
+	Version              string `json:"Version"`
+	ContainerInstanceARN string `json:"ContainerInstanceARN"`
```

Comment:
Given that we’re already in the `Instance` `struct`, having `Instance` as well in the field name sounds redundant.
Would it be confusing to simplify this by:
```suggestion
	Arn string `json:"Arn"`
```

Reply:
This is the response format AWS uses so we can't change the json struct tag